mat/cholesky_test.go

```diff
@@ -72,6 +72,35 @@ func TestCholesky(t *testing.T) {
 	}
 }
 
+func TestCholeskyAt(t *testing.T) {
+	for _, test := range []*SymDense{
+		NewSymDense(3, []float64{
+			53, 59, 37,
+			59, 83, 71,
+			37, 71, 101,
+		}),
+	} {
+		var chol Cholesky
+		ok := chol.Factorize(test)
+		if !ok {
+			t.Errorf("Matrix not positive definite")
+		}
+		n := test.Symmetric()
+		if chol.Symmetric() != n {
+			t.Errorf("Cholesky size does not match")
```

Comment:
This `Errorf` without verbs will get caught by golint.

```
cn := chol.Symmetric()
if cn != n {
    t.Errorf("unexpected Cholesky size: got:%d want:%d", cn, n)
```

Reply:
Done.